packages/lisk-transaction-pool/browsertest/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+	"env": {
+		"mocha": true,
```

Comment:
I don't think we need `browsertest` for transaction-pool.
It's a server-side only feature